docker-compose.yml

```diff
@@ -16,6 +16,38 @@ services:
       timeout: 5s
       retries: 5
 
+  localstack:
+    container_name: feedback-localstack
+    image: localstack/localstack:0.12.18
+  # make the secret ahead of time needed by feedback-api for local testing
+    privileged: true
+    ports:
+      - 4563-4599:4563-4599
+    environment:
+      - SERVICES=secretsmanager
+      - DEFAULT_REGION=eu-west-1
+      - HOSTNAME=localstack
+      - DEBUG=1
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /tmp/localstack:/tmp/localstack
+    healthcheck:
+      test: ["CMD", "curl", "http://localhost:4566/health?reload"]
+
+  aws-cli:
+    container_name: feedback-aws-cli
+    image: amazon/aws-cli:latest
+    stdin_open: true
+    tty: true
+    environment:
+      - AWS_SECRET_ACCESS_KEY=dummy
+      - AWS_ACCESS_KEY_ID=dummy
+        #command: --region="eu-west-1" --endpoint-url="http://localstack:4566" secretsmanager list-secrets
```

Comment:
Is this meant to be commented out?

Reply:
That was old, sorry, good catch :) have now pushed change with it gone :)